pynamodb/tests/data.py

```diff
@@ -466,7 +466,7 @@
                     'N': '31'
                 },
                 'is_dude': {
-                    'N': '1'
+                    'BOOL': True
```

Comment:
i think i may have been using this to test the old legacyboolattr, but if we remove that this is fine

Reply:
there are tests specific to legacy deserialization `TestLegacyBooleanAttribute`